app/Phragile/BurndownChart.php

```diff
@@ -9,13 +9,13 @@ class BurndownChart {
 	private $pointsClosedPerDay = null;
 	private $sprint = null;
 	private $tasks = null;
-	private $phabricator = null;
+	private $transactions = null;
 
-	public function __construct(\Sprint $sprint, TaskList $tasks, PhabricatorAPI $phabricator)
+	public function __construct(\Sprint $sprint, TaskList $tasks, array $transactions)
```

Comment:
\o/

now It's not entirely obvious what type the parameter is now though. I'm assuming it's an array of `Transaction`, though it's good to document this.